crates/telio-model/src/features.rs

```diff
@@ -31,6 +31,8 @@ pub struct Features {
     /// Test environment (natlab) requires binding feature disabled
     /// TODO: Remove it once mac integration tests support the binding mechanism
     pub is_test_env: Option<bool>,
+    /// Controll if IP addresses should be hidden in logs
+    pub hide_ips: Option<bool>,
```

Comment:
Does this really need to be optional? Adding a `#[serde(default)]` will make sure it's false if it's not specified in json and if doing `Features::default()`

Reply:
After writing this I realized I got the logic backwards for this in my head, and it should of course default to true. Personally I would be in favor of still not having it be optional, and either inverting the name or using the smart defaults crate that Michal introduced

Reply:
Good point, it doesn't have to be optional. I kept the name and used the `SmartDefault` to make it default to `true`.